readme.md

```diff
@@ -94,6 +112,8 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Your Answer:**
 
+`app.use()` gets called every time a request is sent.
```

Comment:
app.use iwll be called whenever a request reaches that point of the route chain.